### DISCUSSION TITLE
video call crash

### DISCUSSION THREAD

**Discussion by HamidRezaBashiri** [flutterjanus/flutter_janus_client#75] [Q&A] [answered]
hi thanks in advance for you're helps 🙏
im running the example on two android studio emulator but when im trying to call other user after pressing accept button app crashes the below is error message

> E/flutter (14008): [ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: This widget has been unmounted, so the State no longer has a context (and should be considered defunct).
E/flutter (14008): Consider canceling any active work during "dispose" or using the "mounted" getter to determine if the State is still active.
E/flutter (14008): #0      State.context.<anonymous closure> (package:flutter/src/widgets/framework.dart:930:9)
E/flutter (14008): #1      State.context (package:flutter/src/widgets/framework.dart:936:6)
E/flutter (14008): #2      _VideoCallV2ExampleState.destroy (package:janus_client_example/typed_examples/video_call.dart:228:18)
E/flutter (14008): #3      _VideoCallV2ExampleState.initJanusClient.<anonymous closure> (package:janus_client_example/typed_examples/video_call.dart:183:15)
E/flutter (14008): #4      _VideoCallV2ExampleState.initJanusClient.<anonymous closure> (package:janus_client_example/typed_examples/video_call.dart:147:40)
E/flutter (14008): #5      _rootRunUnary (dart:async/zone.dart:1434:47)
E/flutter (14008): #6      _CustomZone.runUnary (dart:async/zone.dart:1335:19)
E/flutter (14008): #7      _CustomZone.runUnaryGuarded (dart:async/zone.dart:1244:7)
E/flutter (14008): #8      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:341:11)
E/flutter (14008): #9      _BufferingStreamSubscription._add (dart:async/stream_impl.dart:271:7)
E/flutter (14008): #10     _SyncBroadcastStreamController._sendData (dart:async/broadcast_stream_controller.dart:377:25)
E/flutter (14008): #11     _BroadcastStreamController.add (dart:async/broadcast_stream_controller.dart:244:5)
E/flutter (14008): #12     _AsBroadcastStreamController.add (dart:async/broadcast_stream_controller.dart:474:11)
E/flutter (14008): #13     _rootRunUnary (dart:async/zone.dart:1434:47)
E/flutter (14008): #14     _CustomZone.runUnary (dart:async/zone.dart:1335:19)
E/flutter (14008): #15     _CustomZone.runUnaryGuarded (dart:async/zone.dart:1244:7)
E/flutter (14008): #16     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:341:11)
E/flutter (14008): #17     _DelayedData.perform (dart:async/stream_impl.dart:591:14)
E/flutter (14008): #18     _StreamImplEvents.handleNext (dart:async/stream_impl.dart:706:11)
E/flutter (14008): #19     _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:663:7)
E/flutter (14008): #20     _rootRun (dart:async/zone.dart:1418:47)
E/flutter (14008): #21     _CustomZone.run (dart:async/zone.dart:1328:19)
E/flutter (14008): #22     _CustomZone.runGuarded (dart:async/zone.dart:1236:7)
E/flutter (14008): #23     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1276:23)
E/flutter (14008): #24     _rootRun (dart:async/zone.dart:1426:13)
E/flutter (14008): #25     _CustomZone.run (dart:async/zone.dart:1328:19)
E/flutter (14008): #26     _CustomZone.runGuarded (dart:async/zone.dart:1236:7)
E/flutter (14008): #27     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1276:23)
E/flutter (14008): #28     _microtaskLoop (dart:async/schedule_microtask.dart:40:21)
E/flutter (14008): #29     _startMicrotaskLoop (dart:async/schedule_microtask.dart:49:5)
E/flutter (14008):

**Comment by shivanshtalwar0** [comment, marked as answer]
I am aware of this issue i will fix it as soon as possible
